packages/types/src/transaction.ts

```diff
@@ -115,7 +115,8 @@ export interface SamplingContext extends CustomSamplingContext {
   request?: ExtractedNodeRequestData;
 }
 
-export type Measurements = Record<string, { value: number }>;
+export type MeasurementUnit = 'ns' | 'ms' | 's';
```

Comment:
fyi in the [units DACI](https://www.notion.so/sentry/Custom-Measurements-Units-Protocol-7d3752b43bb946bf87d96e401268ca02) we said no validation SDK side, so I might just remove this list in the future when it'll be a lot more units.

Reply:
I'd rather not include unit validation and just make this an alias type for any string.

Reply:
I was not aware of this. Thanks for pointing that out. Updated in dd9f8c80a4a222a0a5750483821977ab52fcf099.

Ad alias type, I decided to just change it to string, no alias. DX wise I find alias types annoying - just tell me that's a string lol